specs/slashtags-derivation.md

```diff
@@ -0,0 +1,74 @@
+# Slashtags derivation
+
+This document describes how Bitcoin wallets and other Slashtags-powered applications can use standard BIP-39 mnemonic words to derive **slashtags** (that is, the main keypairs that are used within the Slashtags protocol). 
+
+
+## Goals
+
+We had three main goals in our design for the slashtag derivation scheme. 
+
+1. Every slashtag should be backed up via a set of BIP-39 mnemonic words. This is the dominant industry standard for backing up a user's Bitcoin funds. So one set of BIP-39 mnemonic words can back up both a user's Bitcoin funds and their slashtags (as well as any further keypairs derived from those slashtags). 
+2. A public key that can be used to receive Bitcoin funds should never be the public key for a slashtag (or a public key in a keypair derived from a slashtag). 
+3. A slashtag should be generated with a semantically relevant name instead of an index number (the latter is the case with Bitcoin keypairs for creating receiving addresses). 
+
+
+## Derivation
+
+### Mnemonic words, master seed, and master node
+
+Following [BIP-39](https://bips.xyz/39), a series of mnemonic words is created (12, 15, 18, 21, or 24 words). These mnemonic words should be stored by the user of the application for the backup of their slashtags and, typically, Bitcoin funds.  
+
+Following BIP-39, this set of mnemonic words is used to produce a 512-bit master seed (also known as a "BIP-39 seed" or simply the "seed"). The master seed derivation procedure allows for increasing security via a password, which typically should be memorized or at least backed up separately from the mnemonic words.  
```

Comment:
Why are we explaining bip39 here?

Reply:
As discussed to make our specs accessible to developers that are new to Bitcoin.